File: 코딩 기초 트레이닝/접미사 배열-p181909/접미사 배열.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181909

function solution(my_string) {
    var answer = [];

    my_string = my_string.split("").reverse().join('');

    for(let i = 1 ; i <= my_string.length ; i++) {
        answer.push(my_string.slice(0, i).split("").reverse().join(''));
    }

    answer.sort();

    return answer;
}

File: 코딩 기초 트레이닝/접미사 배열-p181909/접미사 배열.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181909

def solution(my_string):
    answer = []
    my_string = my_string[::-1]
    for i in range(len(my_string)):
        i = i+1
        answer.append(my_string[:i][::-1])
    answer.sort()

    return answer
